airflow-core/src/airflow/api_fastapi/core_api/app.py

```diff
@@ -55,6 +55,13 @@ def init_views(app: FastAPI) -> None:
 
     templates = Jinja2Templates(directory=directory)
 
+    if dev_mode:
+        app.mount(
+            "/static/i18n",
+            StaticFiles(directory=Path(AIRFLOW_PATH) / "airflow/ui/public/i18n"),
```

Comment:
As there is also the check scripts - should the mount maybe only provide the "locales" subfolder?

Reply:
Your suggestion sounded good - I updated it to mount only the 'locales' subfolder. Thanks!